docs/wallet/extension-wallet/overview.md

```diff
@@ -4,12 +4,17 @@ title: Overview
 sidebar_label: Overview
 ---
 
-Alephium extension wallet helps users to store and manage their assets on the Alephium blockchain. 
-It also enables dApps to securely interact with the wallets, allowing dApp users to sign transactions,
-execute smart contracts, etc right in the browser.
+Alephium extension wallet helps users to store and manage their assets
+such as the native `ALPH` token, other fungible tokens as well as NFTs
+on the Alephium blockchain.  It also enables dApps to securely
+interact with the wallets, allowing dApp users to sign transactions,
+interact with the smart contracts, etc right from the browser.
 
-<img src={require("./media/landing.png").default} alt="Execute Transction Script" width="250" />
+<img src={require("./media/landing.png").default} alt="Execute Transction Script" />
```

Comment:
There is a zoom-in reset button in the screenshot 😅

Reply:
Updated :)